### ISSUE TITLE
Copy and pasting does not work

### ISSUE THREAD

**Issue by catruge** [JonComo/anim#50]
Hello. On Windows and Linux, copy and pasting doesn't work. I suspect this is because the <kbd>ctrl</kbd> key plus <kbd>c</kbd> or <kbd>v</kbd> is prevented when a `Text` instances return `true` from `onkeydown`:
```
if (rtv.keys.ctrl) {
  this.properties[rtv.frame] = transform_props(
    key,
    this.properties[rtv.frame],
  );
  return true;
}
```

**Comment by catruge** [comment]
Hi. I got the clipboard fixed in #51.

**Comment by JonComo** [comment]
Will close this once we get it merged, thanks!
